tests/library/test_metrics.py

```diff
@@ -698,7 +699,19 @@ def test_kendalltau(self):
         outputs = apply_metric(
             metric=metric, predictions=predictions, references=references
         )
-        global_target = 0.81649658092772
+        global_target = 0.6025716662406921
+        self.assertAlmostEqual(global_target, outputs[0]["score"]["global"]["score"])
+
```

Comment:
```suggestion
        global_target = 0.81649658092772
```
Seems like you override kandall tau test by accident

Reply:
you're right. apologies, I added this back in the latest commit! thanks for catching